tools/scripts/deploy.sh

```diff
@@ -6,19 +6,19 @@
 # See LICENSE file for more details.
 ###############################################################
 
-scriptdir="$(cd "${0%/*}"; pwd)"
+scriptdir="$(cd "${0%/*}" || exit; pwd)"
```

Comment:
Just a quick note: the `|| exit` change is enough to make shellcheck happy, but it doesn't handle the error in any way: if `cd` fails `scriptdir` will still be empty, which can still lead to the issues pointed out by [SC2164](https://github.com/koalaman/shellcheck/wiki/SC2164).
However, I don't think we need to handle these kind of issues either, so if the point is only to silence shellcheck, it's easier to just change `;` to `&&`, so:
```
$(cd "${0%/*}" && pwd)
```
The result will be the same: exit if cd fails, otherwise run pwd.

Otherwise it's good that we already address the shellcheck issues we can address, so that we can run it in CI as a required check at some point, so thanks for taking the time to make those changes :-)